net/yazproxy/files/dynamic_lookup-11.patch

```diff
@@ -1,26 +1,31 @@
 Recognize macOS 11 and later.
 https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44605
 https://github.com/indexdata/yazproxy/issues/21
---- configure.orig	2024-05-23 14:09:36.000000000 -0500
-+++ configure	2024-05-23 14:10:02.000000000 -0500
-@@ -7923,16 +7923,11 @@
-       _lt_dar_allow_undefined='${wl}-undefined ${wl}suppress' ;;
+--- configure	2022-08-18 16:32:05
++++ configure	2024-06-06 12:59:04
+@@ -9083,18 +9083,13 @@
+       _lt_dar_allow_undefined='$wl-undefined ${wl}suppress' ;;
      darwin1.*)
-       _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
+       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
 -    darwin*) # darwin 5.x on
 -      # if running on 10.5 or later, the deployment target defaults
 -      # to the OS version, if on x86, and 10.4, the deployment
 -      # target defaults to 10.4. Don't you love it?
 -      case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
--	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
--	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
--	10.[012]*)
+-	10.0,*86*-darwin8*|10.0,*-darwin[912]*)
+-	  _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
+-	10.[012][,.]*)
+-	  _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
+-	10.*|11.*)
+-	  _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
+-      esac
 +    darwin*)
 +      case ${MACOSX_DEPLOYMENT_TARGET},$host in
-+	10.[012],*|,*powerpc*)
- 	  _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
--	10.*)
-+	*)
- 	  _lt_dar_allow_undefined='${wl}-undefined ${wl}dynamic_lookup' ;;
-       esac
++    10.[012],*|,*powerpc*-darwin8*)
```

Comment:
It makes no difference for MacPorts since it always sets `MACOSX_DEPLOYMENT_TARGET` and since MacPorts doesn't support Darwin < 8 but for consistency with the code produced by libtool 2.4.7 and later this should read:
```suggestion
+    10.[012],*|,*powerpc*-darwin[5-8]*)
```
See https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44605#20.